lib/Default/MySqlDatabase.class.php

```diff
@@ -9,16 +9,29 @@ abstract class MySqlDatabase {
 	protected static $selectedDbName;
 	protected $dbResult = null;
 	protected $dbRecord = null;
-	
+
 	public function __construct($dbName) {
 		if (!self::$dbConn) {
 			// Set the mysqli driver to raise exceptions on errors
 			if (!mysqli_report(MYSQLI_REPORT_ERROR | MYSQLI_REPORT_STRICT)) {
 				$this->error('Failed to enable mysqli error reporting');
 			}
 
-			self::$dbConn = new mysqli(self::$host, self::$user, self::$password,
-			                           $dbName, self::$port, self::$socket);
+            $host = self::$host;
```

Comment:
It looks like the indentation level for these 4 lines is too small.

Reply:
Thanks, I think that was remnants from the spaces formatting